08_voting/index.tex

```diff
@@ -1 +1,101 @@
 \chapter{Voting}
+\section{Voting Scenarios}
+\label{sec:VotingScenarios}
+
+Within the context of the coursework, robust voting systems are designed for deciding the outcome of collective actions (e.g. elections for legislative, executive and judicial branches in IIGO).
```

Comment:
technically for academic writing we should be using the passive voice. Suggested re-write: 

Within the context of the coursework, robust voting systems were designed to determine the outcome of collective actions (e.g. elections for legislative, executive and judicial branches in IIGO).

Reply:
so your suggestion is "are" --> "were" ??
"are designed" is also passive voice. I don't understand your point @yp717

Reply:
designed for --> designed to .
ok I agree with this.

Reply:
the sentence 5 is already _passive voice_ . so your point is not right, I think @yp717

Reply:
changed